### DISCUSSION TITLE
mutate( var = 1:n() )

### DISCUSSION THREAD

**Discussion by mike-lawrence** [markfairbanks/tidypolars#237] [Q&A]
New to tidypolars and I'm looking for the equivalent to `df %>% mutate( var = 1:n() )`.

**Comment by markfairbanks** [comment]
In this case you can just use `tp.row_number()`. It would probably be worth adding a `tp.seq()` that allowed other start points than 1.

```python
import tidypolars as tp
from tidypolars import col

df = tp.Tibble({"x": range(3), "y": ["a", "a", "b"]})

df.mutate(var = tp.row_number())
```
```
shape: (3, 3)
┌─────┬─────┬─────┐
│ x   ┆ y   ┆ var │
│ --- ┆ --- ┆ --- │
│ i64 ┆ str ┆ i64 │
╞═════╪═════╪═════╡
│ 0   ┆ a   ┆ 1   │
│ 1   ┆ a   ┆ 2   │
│ 2   ┆ b   ┆ 3   │
└─────┴─────┴─────┘
```

However it's worth noting that `tidypolars` is a semi-dead project. `polars` was pushing breaking changes too often for me to keep up. At this point it's probably just safer to use `polars` directly and locking to a specific version.
